odp_sdk/tabular_storage_client.py

```diff
@@ -386,6 +388,8 @@ def _select_page(
             result = PaginatedSelectResultSet(**response.json())
         except JSONDecodeError:
             data = list(iter(NdJsonParser(response.text)))
+            data = convert_geometry(data, result_geometry)
```

Comment:
Why does this happen in the exception-handler?

Reply:
the try catch is first trying to parse the response as a plain json, if it cant it catches an exception and tries the ndjson parser